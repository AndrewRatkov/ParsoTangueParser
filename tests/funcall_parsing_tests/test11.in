h(f(3) + (g() == g()), g() * f(23))