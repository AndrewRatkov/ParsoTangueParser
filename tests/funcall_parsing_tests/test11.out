[VOID h]------->[INT,+ ]------->[INT f]------->[INT,3]
    |               |
    |               +---------->[INT,==]------->[STR g]
    |                               |
    |                               +---------->[STR g]
    |
    +---------->[STR,* ]------->[STR g]
                    |
                    +---------->[INT f]------->[INT,23]
